editor/plugins/theme_editor_plugin.cpp

```diff
@@ -3001,7 +3001,7 @@ ThemeEditor::ThemeEditor() {
 	add_child(top_menu);
 
 	theme_name = memnew(Label);
-	theme_name->set_text(TTR("Theme") + ": ");
+	theme_name->set_text(TTR("Theme:"));
```

Comment:
This loses the space but maybe it doesn't matter here?

Reply:
Yeah, I removed it on purpose. We don't append the name, we only replace it (the other changed line in this file is where we do it), so it shouldn't break anything. The spaces was a leftover for similarity.